classes/ETL/Ingestor/StructuredFileIngestor.php

```diff
@@ -346,12 +346,12 @@ function ($k, $v) {
             }
 
             $this->destinationHandle->commit();
-        } catch (PDOException $e) {
+        } catch (Exception $e) {
+            $this->destinationHandle->rollback();
             $this->logAndThrowException(
                 "Error committing transaction. Rolling back transactions.",
```

Comment:
```suggestion
                "Error committing transaction. Rolling back transaction.",
```
It's just one transaction that gets rolled back, right?

Reply:
@aaronweeden Yeah, it is.